entries/SWC-135.md

```diff
@@ -0,0 +1,17 @@
+# Title
+Statements With No Effects
+
+## Relationships
+[CWE-561: Dead Code](https://cwe.mitre.org/data/definitions/561.html)
+
+## Description
+In Solidity, it's possible to write statements that have no side effects. Currently, the solidity compiler will not return a warning for side-effect-free statements. This can lead to statements going unnoticed, causing a function to proceed without properly performing an intended action.
```

Comment:
In which language is it not possible to do that, just curious.

Reply:
I'm not sure of any in particular.